control-plane/csi-driver/src/bin/node/nodeplugin_grpc.rs

```diff
@@ -46,18 +50,50 @@ impl NodePlugin for NodePluginSvc {
     ) -> Result<Response<FreezeFsReply>, Status> {
         let volume_id = request.into_inner().volume_id;
         debug!("freeze_fs({})", volume_id);
-        freeze_volume(&volume_id).await?;
-        Ok(Response::new(FreezeFsReply {}))
+        let _guard = VolumeFsfreezeGuard::new_str(&volume_id)?;
+        let output = Command::new("csi-node")
+            .arg("--node-name")
+            .arg("dummy")
+            .arg("fsfreeze")
+            .arg("-v")
+            .arg(volume_id.clone())
+            .output()
+            .await
+            .map_err(|error| Status::aborted(error.to_string()))?;
```

Comment:
So an error from freeze/unfreeze is always converted to aborted?

Reply:
Have refactored it, it would now contain the Errno and accordingly we can return specific errors.